sysbuild/CMakeLists.txt

```diff
@@ -490,6 +490,20 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
   if(SB_CONFIG_SDP)
     include_sdp()
   endif()
+
+  foreach(config APPROTECT_USE_UICR APPROTECT_LOCK APPROTECT_USER_HANDLING
+  SECURE_APPROTECT_USE_UICR SECURE_APPROTECT_USER_HANDLING SECURE_APPROTECT_LOCK)
+    if(SB_CONFIG_${config})
+      if(SB_CONFIG_BOOTLOADER_MCUBOOT)
+        set_config_bool(mcuboot CONFIG_NRF_${config} y)
+      endif()
+      if(SB_CONFIG_SECURE_BOOT_APPCORE)
+        set_config_bool(b0 CONFIG_NRF_${config} y)
+      endif()
+      set_config_bool(${DEFAULT_IMAGE} CONFIG_NRF_${config} y)
+    endif()
+  endforeach()
```

Comment:
this is not setting it for all images, should it set it for all or not? Including network core?

Reply:
I see no resolution or comment to this despite it being marked as resolved?

Reply:
The comment is addressed in the lines below this.
 https://github.com/nrfconnect/sdk-nrf/pull/17836/files#diff-f5dd191b79215aef1d7348f1d5b9c4461b672860a09f95b5d7ed8c3d9f73a42dR506-R516?

Reply:
Network core does not have secure versions of these Kconfigs, so setting them would cause warnings.

Reply:
@nordicjm: Please advice, if you want something else and whether the nits should be addressed. I am not sure if I have time to run the pipelines again successfully.